packages/flutter/lib/src/material/overscroll_indicator.dart

```diff
@@ -132,7 +132,9 @@ class _OverscrollIndicatorState extends State<OverscrollIndicator> {
   void _hide() {
     _hideTimer?.cancel();
     _hideTimer = null;
-    _extentAnimation.reverse();
+    if (!_extentAnimation.isAnimating) {
```

Comment:
What we're guarding against here is if the indicator is already animating away when _hide is called? I think that his would be a good place for a comment.